src/Tribe/Commerce/Warnings.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Conditional Warnings on Ticket Features
+ *
+ * Class Tribe__Tickets__Commerce__Warnings
+ *
+ * @since TBD
+ *
+ */
+class Tribe__Tickets__Commerce__Warnings {
+
+	public function hook() {
+		add_action( 'tribe_events_tickets_new_ticket_buttons', [ $this, 'recurring_event_warning_msg' ] );
```

Comment:
:no_entry_sign: Missing doc comment for function hook()

<sub>hash: a9251eb177fd9c190ecd21469cbd9d29</sub>